db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -843,8 +850,11 @@ void DBImpl::NotifyOnFlushBegin(ColumnFamilyData* cfd, FileMetaData* file_meta,
     info.job_id = job_id;
     info.triggered_writes_slowdown = triggered_writes_slowdown;
     info.triggered_writes_stop = triggered_writes_stop;
-    info.smallest_seqno = file_meta->fd.smallest_seqno;
-    info.largest_seqno = file_meta->fd.largest_seqno;
+    // This sequence number is actually smaller than or equal to the sequence
+    // number
```

Comment:
Wrong line break.

Reply:
It was formatted by clang-format

Reply:
It's possible for it to break "sequence" and "number" due to 80 char limit. But "number" and "of" won't be broken.